pkg/autohealer/healer.go

```diff
@@ -428,3 +432,74 @@ func (h *Healer) updateSilence(e *common.Etcd) {
 		h.silences[sid] = &s
 	}
 }
+
+// syncSilencesFromBackend queries silences from the metric backend. Only Prometheus Alertmanager
+// is supported at this time.
+func (h *Healer) syncSilencesFromBackend(ctx context.Context, e *common.Etcd) {
+	if h.backend.GetType() != model.PrometheusType || !h.SyncSilences {
+		level.Info(h.logger).Log("msg", "Skip sync silences",
```

Comment:
Move to debug or just omit it. Just notify when users create healer that their monitor backend does not support sync silences.